avm/src/daemon/daemon.rs

```diff
@@ -1,17 +1,20 @@
 use std::collections::HashMap;
 use std::env;
 use std::error::Error;
-use std::fs::read;
+use std::fs::{read, write};
 use std::io::Read;
 
+use anycloud::common::{get_app_tar_gz_b64, get_dockerfile_b64};
 use anycloud::deploy;
 use anycloud::{error, CLUSTER_ID};
 use base64;
 use byteorder::{LittleEndian, ReadBytesExt};
 use flate2::read::GzDecoder;
+use lazy_static::lazy_static;
 use once_cell::sync::OnceCell;
 use serde::{Deserialize, Serialize};
 use serde_json::{json, Value};
+#[allow(unused_imports)]
```

Comment:
Why is this line needed? You use `Command` below in the Linux `get_private_ip` function. Is the warning showing up on OSX and Windows? @cdmistman would `#[cfg(...` guarding this allow work?

Reply:
good point - yes, adding the `#[cfg(...` here would work but it might be better to just directly import in the function?
```rust
#[cfg(target_os = "linux")]
async fn get_private_ip() -> DaemonResult<String> {
  use tokio::process::Command;
  ...
}
```

Reply:
Yes, I was having the warning on OSX. I'll use `tokio::process::Command(...` directly.